test/route-beutifyLocation.js

```diff
@@ -1,34 +1,9 @@
-var PushStateTree = require('../src/pushStateTree');
+const PushStateTree = require('../src/pushStateTree');
+import cleanHistoryAPI from './helper/cleanHistoryAPI';
 
 describe('PushStateTree beutifyLocation should', function() {
```

Comment:
nab: as the 'should'  have been removed from describe and put in the 'it' function in route-basePath.js, wouldn't be better if the other files followed the same pattern?